{"name":"verifySuccessfulRegisteration","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [My Account] but found [Register]","trace":"java.lang.AssertionError: expected [My Account] but found [Register]\r\n\tat org.testng.Assert.fail(Assert.java:97)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:575)\r\n\tat org.testng.Assert.assertEquals(Assert.java:585)\r\n\tat completeScenarios.NPwdLimit.verifySuccessfulRegisteration(NPwdLimit.java:59)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:766)\r\n\tat org.testng.TestRunner.run(TestRunner.java:587)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1109)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1039)\r\n\tat org.testng.TestNG.run(TestNG.java:1007)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"Test all fields are valid and there isn't a limit on the password field {3 char}. 3. Verify unsuccessful registeration...","steps":[],"attachments":[{"name":"saveFailureScreenShot","source":"88cdd4b4-b585-4a00-860a-c97de0d22934-attachment"},{"name":"verifySuccessfulRegisteration failed and screenshot taken!","source":"a295d3dd-54f5-4ae5-a342-f83e3825402f-attachment","type":"text/plain"}],"parameters":[],"start":1624744225998,"stop":1624744226155,"uuid":"3d4b2857-31cd-44c0-a870-3d9de57d3a42","historyId":"2ef8d9778092875b8b0205a511a536bb","fullName":"completeScenarios.NPwdLimit.verifySuccessfulRegisteration","labels":[{"name":"package","value":"completeScenarios.NPwdLimit"},{"name":"testClass","value":"completeScenarios.NPwdLimit"},{"name":"testMethod","value":"verifySuccessfulRegisteration"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"completeScenarios.NPwdLimit"},{"name":"host","value":"Hagar-Elsayed"},{"name":"thread","value":"24336@Hagar-Elsayed.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}